draft-ietf-tls-tls13.md

```diff
@@ -1567,8 +1567,8 @@ cipher_suites
   the server does not recognize, support, or wish to use, the server
   MUST ignore those cipher suites, and process the remaining ones as
   usual. Values are defined in {{cipher-suites}}. If the client is
-  attempting a PSK key establishment, it MUST only advertise the single
-  cipher suite associated with the PSK.
+  attempting a PSK key establishment, it MUST advertise at least one
+  cipher suite containing a Hash associated with the PSK.
```

Comment:
That section is already pretty loaded though. I still think the mention would be better in that cipher suite definition of the client hello. I tried a change to a SHOULD, if this is really not satisfying I can revert this commit.